### PR TITLE
wizard step fixes

### PR DESCRIPTION
**Pull request by aferd**

### CODE REVIEW

File: src/pages/Integrations/Create/helpers.tsx [hunk 1]
```diff
@@ -54,9 +54,13 @@ export const nextDetailsStep = (values) => {
     return 'slack-details';
   } else if (
     values['integration-type'] === IntegrationType.TEAMS ||
-    IntegrationType.GOOGLE_CHAT
+    values['integration-type'] === IntegrationType.GOOGLE_CHAT
   ) {
     return 'teams-gchat-details';
+  } else if (values['integration-type'] === IntegrationType.SPLUNK) {
```
Comment (karelhala) [inline src/pages/Integrations/Create/helpers.tsx]: Ideally we'll utilize DDF conditional steps https://www.data-driven-forms.org/examples/custom-wizard#heading-customwizard

File: src/pages/Integrations/Create/IntegrationsStep.tsx [hunk 2]
```diff
@@ -2,18 +2,25 @@ import {
...
-  nextStep: ({ values }) => nextDetailsStep(values),
+  name: 'integration-type',
+  nextStep: {
+    when: 'integration-type',
+    stepMapper: {
+      'camel:slack': 'slack-details',
+      'camel:google_chat': 'teams-gchat-details',
+      'camel:teams': 'teams-gchat-details',
+      'camel:splunk': 'splunk-details',
+      'camel:servicenow': 'service-now-details',
```
Comment (karelhala) [inline src/pages/Integrations/Create/IntegrationsStep.tsx]: Please import the `IntegrationType` and use for instance `[IntegrationType.SLACK]: 'slack-details'` that way if we were to change the type we don't have to change it here as well.

**Review by karelhala — COMMENTED** [review]
Looking good! Just minor adjustment of `stepMapper` and we are golden.

**Review by fhlavac — COMMENTED** [review]
Apart from Karel's comment looks good! Thank you!

**Review by fhlavac — APPROVED** [review]

**Pull request merged**